build/README.md

```diff
@@ -0,0 +1,64 @@
+# Building Rook
```

Comment:
actually, should this content be merged into the INSTALL.md file in the root of the repo? We already have some build instructions there